examples/solidity/basic/balance.sol

```diff
@@ -0,0 +1,11 @@
+contract C {
+  constructor() public {
+      //msg.sender.transfer(0);
```

Comment:
let's either remove this, include it, or document why it's commented out

Reply:
Fixed.